src/js/views/awardsRecords.js

```diff
@@ -120,7 +105,7 @@ function getPlayerAwards(p, awardType) {
     const formatYear = year => {
         return Object.keys(year).map(k => {
             const years = helpers.yearRanges(year[k].map(y => y.season)).join(', ');
-            return `${k} <small>(${years})</small>`;
+            return `${k} (${years})`;
```

Comment:
As mentioned in the pull comment, I only removed it because I couldn't figure out how to make it work passing in a React component.  Is there an example where you're doing this?  I'm not really understanding what you're saying in the second approach, we need to pass in an array of objects with a data property containing a field per field on the table for the rows argument to DataTable right?